doc/source/dev/development_gitpod.rst

```diff
@@ -135,12 +133,12 @@ before working on your contributions.
 When using Gitpod, git is pre configured for you:
 
 #. You do not need to configure your git username, and email as this should be 
-#. done for you as you authenticated through GitHub. You can check the git 
-#. configuration with the command ``git config --list`` in your terminal.
+   done for you as you authenticated through GitHub. You can check the git 
+   configuration with the command ``git config --list`` in your terminal.
 #. As you started your workspace from your own NumPy fork, you will by default 
-#. have both "upstream "and "origin "added as remotes. You can verify this by 
-#. typing ``git remote`` on your terminal or by clicking on the **branch name** 
-#. on the status bar (see image below).
+   have both "upstream "and "origin "added as remotes. You can verify this by 
+   typing ``git remote`` on your terminal or by clicking on the **branch name** 
+   on the status bar (see image below).
```

Comment:
Is it my eyes or are there two "1" labels and no "6" in the image? Maybe the labels could be a bit larger?